test/fixtures/table-attr.html

```diff
@@ -5,7 +5,7 @@
         background-color: red;
         float: left;
         width: 100%;
-        height: 500px;
+        height: 500PX;
```

Comment:
Why is this change needed?

Reply:
The regex to replace px values is case insensitive. This tests that `PX` which is valid CSS will also be removed.